index.html

```diff
@@ -1,15 +1,87 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Is there a reason this got changed to lowercase, or was it a side effect of something else? 
```suggestion
<!DOCTYPE html>
```